Core/Helper.php

```diff
@@ -21,6 +23,8 @@
  */
 class Helper
 {
+    const MODULE_ID = 'wdoxidee';
```

Comment:
Can u use this constant now in metadata.php, PaymentGateway.php and wirecard_credit_card_fields.tpl the string is there hard coded. So if a change accrues its only needed to update one const.

Reply:
refactored